### PR TITLE
use alternatives command on javac as well for maven

### PR DESCRIPTION
**Pull request by gabemontero**
Fixes https://github.com/openshift/jenkins/issues/522

@openshift/sig-developer-experience ptal

### CODE REVIEW

**Comment by gabemontero** [comment]
[test]

**Comment by bparees** [comment]
should probably do it in jenkins master too, so we're consistent.

**Comment by bparees** [comment]
(unless we don't install an SDK in that image?)

**Comment by gabemontero** [comment]
javac does exist there ... will update master image

**Comment by gabemontero** [comment]
master update pushed

File: 2/contrib/s2i/run [hunk 1]
```diff
@@ -151,12 +151,14 @@ CONTAINER_MEMORY_IN_MB=$((CONTAINER_MEMORY_IN_BYTES/2**20))
...
   alternatives --set java $(alternatives --display java | awk '/family.*x86_64/ { print $1; }')
+  alternatives --set javac $(alternatives --display javac | awk '/family.*x86_64/ { print $1; }')
 else
   alternatives --set java $(alternatives --display java | awk '/family.*i386/ { print $1; }')
+  alternatives --set javac $(alternatives --display javac | awk '/family.*i386/ { print $1; }')
   export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
 fi
 
-echo "OPENSHIFT_JENKINS_JVM_ARCH='${OPENSHIFT_JENKINS_JVM_ARCH}', CONTAINER_MEMORY_IN_MB='${CONTAINER_MEMORY_IN_MB}', using $(readlink /etc/alternatives/java)"
+echo "OPENSHIFT_JENKINS_JVM_ARCH='${OPENSHIFT_JENKINS_JVM_ARCH}', CONTAINER_MEMORY_IN_MB='${CONTAINER_MEMORY_IN_MB}', using $(readlink /etc/alternatives/java) and $(readlink /etc/alternatives/java)"
```
Comment (bparees) [inline 2/contrib/s2i/run]: did you intend for this to print two different values? both are referencing java right now, not javac.

Comment (gabemontero) [reply 2/contrib/s2i/run]: yeah the second should be javac ... will update in a moment

**Comment by gabemontero** [comment]
echo fixed

**Comment by bparees** [comment]
[merge]

**Comment by gabemontero** [comment]
[merge]

**Pull request merged**